support/utils/configs.go

```diff
@@ -6,10 +6,12 @@ import (
 	"log"
 	"reflect"
 	"strings"
+
+	"github.com/interstellar/kelp/support/logger"
 )
 
 // CheckConfigError checks configs for errors, crashes app if there's an error
-func CheckConfigError(cfg fmt.Stringer, e error, filename string) {
+func CheckConfigError(l logger.Logger, cfg fmt.Stringer, e error, filename string) {
```

Comment:
this can use utils.utilsLogger instead of threading through these functions (utilsLogger will be private to utils package with the global variable usage describe above)

Reply:
done